src/runtime/minrpc/minrpc_server.h

```diff
@@ -28,14 +28,23 @@
 #ifndef TVM_RUNTIME_MINRPC_MINRPC_SERVER_H_
 #define TVM_RUNTIME_MINRPC_MINRPC_SERVER_H_
 
+#ifndef DMLC_LITTLE_ENDIAN
 #define DMLC_LITTLE_ENDIAN 1
+#endif
+
 #include <string.h>
 #include <tvm/runtime/c_runtime_api.h>
 
+#include <memory>
+#include <utility>
+
 #include "../../support/generic_arena.h"
+#include "minrpc_interfaces.h"
 #include "rpc_reference.h"
 
 /*! \brief Whether or not to enable glog style DLOG */
+#define TVM_MINRPC_ENABLE_LOGGING 1
+
 #ifndef TVM_MINRPC_ENABLE_LOGGING
 #define TVM_MINRPC_ENABLE_LOGGING 0
```

Comment:
is this needed, since it's #define'd to 1 in previous line?

Reply:
Thanks for the comment. it's fixed.